server/src/test/java/org/apache/druid/server/coordinator/TestDruidCoordinatorConfig.java

```diff
@@ -252,4 +223,333 @@ public boolean getCoordinatorKillIgnoreDurationToRetain()
+    private static final Duration DEFAULT_COORDINATOR_SUPERVISOR_KILL_DURATION_TO_RETAIN = new Duration("PT7776000s");
+    private static final Duration DEFAULT_COORDINATOR_COMPACTION_KILL_PERIOD = new Duration("PT86400s");
+    private static final Duration DEFAULT_COORDINATOR_RULE_KILL_PERIOD = new Duration("PT86400s");
+    private static final Duration DEFAULT_COORDINATOR_RULE_KILL_DURATION_TO_RETAIN = new Duration("PT7776000s");
+    private static final Duration DEFAULT_COORDINATOR_DATASOURCE_KILL_PERIOD = new Duration("PT86400s");
+    private static final Duration DEFAULT_COORDINATOR_DATASOURCE_KILL_DURATION_TO_RETAIN = new Duration("PT7776000s");
+    private static final Duration DEFAULT_LOAD_TIMEOUT_DELAY = new Duration(15 * 60 * 1000);
+    private static final Duration DEFAULT_LOAD_QUEUE_PEON_REPEAT_DELAY = Duration.millis(50);
+    private static final String DEFAULT_LOAD_QUEUE_PEON_TYPE = "curator";
+    private static final int DEFAULT_CURATOR_LOAD_QUEUE_PEON_NUM_CALLBACK_THREADS = 2;
+    private static final Duration DEFAULT_HTTP_LOAD_QUEUE_PEON_REPEAT_DELAY = Duration.millis(60000);
+    private static final Duration DEFAULT_HTTP_LOAD_QUEUE_PEON_HOST_TIMEOUT = Duration.millis(300000);
+    private static final int DEFAULT_HTTP_LOAD_QUEUE_PEON_BATCH_SIZE = 1;
+    private static final boolean DEFAULT_COMPACTION_SKIP_LOCKED_INTERVALS = true;
+    private static final Duration DEFAULT_COORDINATOR_AUDIT_KILL_PERIOD = new Duration("PT86400s");
+    private static final Duration DEFAULT_COORDINATOR_AUTIT_KILL_DURATION_TO_RETAIN = new Duration("PT7776000s");
+
+
+    private Duration coordinatorStartDelay;
+    private Duration coordinatorPeriod;
+    private Duration coordinatorIndexingPeriod;
+    private Duration metadataStoreManagementPeriod;
+    private Duration coordinatorKillPeriod;
+    private Duration coordinatorKillDurationToRetain;
+    private Boolean coordinatorKillIgnoreDurationToRetain;
+    private Integer coordinatorKillMaxSegments;
+    private Duration coordinatorSupervisorKillPeriod;
+    private Duration coordinatorSupervisorKillDurationToRetain;
+    private Duration coordinatorCompactionKillPeriod;
+    private Duration coordinatorRuleKillPeriod;
+    private Duration coordinatorRuleKillDurationToRetain;
+    private Duration coordinatorDatasourceKillPeriod;
+    private Duration coordinatorDatasourceKillDurationToRetain;
+    private Duration loadTimeoutDelay;
+    private Duration loadQueuePeonRepeatDelay;
+    private String loadQueuePeonType;
+    private Duration httpLoadQueuePeonRepeatDelay;
+    private Integer curatorLoadQueuePeonNumCallbackThreads;
+    private Duration httpLoadQueuePeonHostTimeout;
+    private Integer httpLoadQueuePeonBatchSize;
+    private Boolean compactionSkippedLockedIntervals;
+    private Duration coordinatorAuditKillPeriod;
+    private Duration coordinatorAuditKillDurationToRetain;
+
+    public Builder()
+    {
+    }
+
+    public Builder(
```

Comment:
Where is this constructor used?

Reply:
ya, that is unused. I removed. If needed in the future it can be easily created with code gen